src/tests/Interop/IDynamicInterfaceCastable/Program.cs

```diff
@@ -81,6 +88,8 @@ int ITest.CallImplemented(ImplementationToCall toCall)
                     return GetNumberPrivate();
                 case ImplementationToCall.InterfaceStatic:
                     return GetNumberStatic();
+                case ImplementationToCall.ImplInterfacePublic:
```

Comment:
> `case ImplementationToCall.ImplInterfacePublic:`

The implementation is `private` right?

Reply:
By default, interface methods are public. That's why this causes an issue in comparison to the test case with a private method on the impl interface.

Reply:
Wait a minute. I thought for default interface implementations by default they were private? I must be misremembering the feature review. Okay, There is no value in relying on the default. Let's be explicit and use the `public` keyword instead.

Reply:
By default, the default implemented method can only be called on an expression of the interface type. That's what caught us off-guard in C#/WinRT which is why I added the test without the modifier. I'll add the keyword.